src/track.rs

```diff
@@ -155,10 +159,14 @@ impl Client {
             ..Default::default()
         }).await.ok()?.into_inner().status?;
 
+        let mut labels = s.labels.clone();
+
         Some(Container {
             id:    s.id.clone(),
             name:  s.metadata?.name.clone(),
             image: s.image?.image.clone(),
+            namespace: labels.remove("io.kubernetes.pod.namespace").clone().unwrap_or_default(),
+            pod: labels.remove("io.kubernetes.pod.name").clone().unwrap_or_default(),
```

Comment:
HashMap.remove() returns the value itself, not a reference, so there's no need to clone after a call to remove. But perhaps `labels.get(...).clone()` makes the intent a little clearer.